### PR TITLE
Octokit setup

### PR DESCRIPTION
**Pull request by Louis-Stekhoven-Smith**
A basic display of how to use octokit to get the data we need
Added  in controller organisations 

Risks 
committed an SSL cert in controllers/config to stop bug. Not super secure though 😄

### CODE REVIEW

File: app/controllers/about_page_controller.rb [hunk 1]
```diff
@@ -3,7 +3,6 @@ class AboutPageController < ApplicationController
   def project_contributors
     @contributors ||= Octokit.contributors('rmit-programming-club/network-vis')
   end
-
```
Comment (thundergolfer) [inline app/controllers/about_page_controller.rb]: Give me back my whitespace Louis I swear

**Review by thundergolfer — APPROVED** [review]
👍 on restoring that line-break. Thanks for adding a test.

File: app/controllers/about_page_controller.rb (hunk 1 above)
Comment (Pancrisp) [inline app/controllers/about_page_controller.rb]: Stop discriminating against final newlines!

**Review by Pancrisp — APPROVED** [review]

**Comment by Louis-Stekhoven-Smith** [comment]
@thundergolfer  @Pancrisp 

Please give this PR a thumbs up again as PR has changed a bit since you guys approved

**Review by thundergolfer — APPROVED** [review]
This is good.

**Review by Pancrisp — APPROVED** [review]
Looks good.

**Pull request merged**